koku/masu/util/aws/common.py

```diff
@@ -901,7 +901,6 @@ def match_openshift_resources_and_labels(data_frame, cluster_topologies, matched
     data_frame["special_case_tag_matched"] = False
     tags = data_frame["resourcetags"]
     if not tags.eq("").all():
-        tags = tags.str.lower()
```

Comment:
Don't we still need to do this for the matching we are doing within this `if` statement? I'd presume there is a reason this is here originally. We probably just need to rename the variable instead of overwriting `tags`.
```
        ocp_tags = tags.str.lower()
        special_case_tag_matched = ocp_tags.str.contains(
```

Reply:
That is a fabulous point.